docs/book/src/developer/providers/bootstrap.md

```diff
@@ -28,6 +28,61 @@ Note: because the `dataSecretName` is part of `status`, this value must be deter
 `Cluster`, `Machine`, and/or bootstrap resource. If the name is randomly generated, it is not always possible to move
 the resource and its associated secret from one management cluster to another.
 
+### BootstrapTemplate Resources
+
+For a given Bootstrap resource, you should also add a corresponding BootstrapTemplate resource:
+
+``` go
+// PhippyBootstrapConfigTemplateSpec defines the desired state of PhippyBootstrapConfigTemplate.
+type PhippyBootstrapConfigTemplateSpec struct {
+	Template PhippyBootstrapTemplateResources `json:"template"`
+}
+
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:path=phippybootstrapconfigtemplates,scope=Namespaced,categories=cluster-api,shortName=pbt
```

Comment:
```suggestion
// +kubebuilder:resource:path=phippybootstrapconfigtemplates,scope=Namespaced,categories=cluster-api,shortName=pbct
```
nit, but not sure. This would follow the pattern that every capital letter gets a character in the short name